### PR TITLE
Add constructor with request field bunq/sdk_java#50

### PR DESCRIPTION
**Pull request by OGKevin**
References #50

### CODE REVIEW

**Comment by OGKevin** [comment]
@patrickdw1991 please 👀

File: src/main/java/com/bunq/sdk/model/core/BunqModel.java [hunk 1]
```diff
@@ -30,6 +31,16 @@
    */
   private static final int INDEX_FIRST = 0;
 
+  /**
+   * String format consents.
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/model/core/BunqModel.java]: I guess it should say contents

File: src/main/java/com/bunq/sdk/model/core/BunqModel.java [hunk 2]
```diff
@@ -30,6 +31,16 @@
...
+   */
+  private static final String STRING_EMPTY = "";
+
+  /**
+   * Regex constatns.
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/model/core/BunqModel.java]: Constants

File: src/main/java/com/bunq/sdk/model/core/BunqModel.java [hunk 3]
```diff
@@ -179,4 +190,11 @@ protected static Integer determineMonetaryAccountId(Integer id) {
       return id;
     }
   }
+
+  protected static byte[] determineRequestByte(HashMap<String, Object> requestMap) {
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/model/core/BunqModel.java]: Not just one byte right?

File: src/main/java/com/bunq/sdk/model/core/BunqModel.java (hunk 1 above)
Comment (OGKevin) [reply src/main/java/com/bunq/sdk/model/core/BunqModel.java]: Constants xD :D

File: src/main/java/com/bunq/sdk/model/core/BunqModel.java (hunk 3 above)
Comment (OGKevin) [reply src/main/java/com/bunq/sdk/model/core/BunqModel.java]: fair enough.

**Comment by OGKevin** [comment]
@patrickdw1991 pushed

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**